src/Functions/bitmaskToList.cpp

```diff
@@ -0,0 +1,133 @@
+#include <Functions/FunctionFactory.h>
+#include <Functions/IFunction.h>
+#include <Columns/ColumnString.h>
+#include <Columns/ColumnVector.h>
+#include <DataTypes/DataTypeString.h>
+#include <IO/WriteBufferFromVector.h>
+#include <IO/WriteHelpers.h>
+
+
+namespace DB
+{
+
+namespace ErrorCodes
+{
+    extern const int ILLEGAL_TYPE_OF_ARGUMENT;
+    extern const int ILLEGAL_COLUMN;
+}
+
+
+/** Function for an unusual conversion to a string:
+  *
+  * bitmaskToList - takes an integer - a bitmask, returns a string of degrees of 2 separated by a comma.
+  *                 for example, bitmaskToList(50) = '2,16,32'
```

Comment:
Maybe we can return `Array(UInt8)`?

Reply:
Oops, this is not my code, it seems something went wrong while resolving conflicts... I will look through changes again